### ISSUE TITLE
Initial pen contact drawn as separate stroke (Win Ink related)

### ISSUE THREAD

**Issue by manongjohn** [opentoonz/opentoonz#2280]
### Issue Summary
With Windows Ink Support enabled in OT and drawing a line with a pen, the initial contact point is drawn as a separate stroke.  May not notice it until you undo the drawn line

This does not happen when Windows Ink Support is not enabled nor does it happen when drawing with a mouse.

### Screenshot or Video Reference
![pen issue](https://user-images.githubusercontent.com/19245851/45198564-2b0bbb00-b235-11e8-9598-3a3438aeda6f.gif)

### Steps to Reproduce
1) If not already done, enable Windows Ink Support in OT preferences and restart OT
2) With a tablet pen draw a line on any image type (more obvious on Vector/Toonz Raster) -> Line is drawn
3) Undo -> Line is removed but a dot remains where the line started

Note: History window records 2 vector strokes created when done drawing the line.

### System Information
* **OpenToonz Version**: 1.2.1 nightly from 9/7
* **Operating System**:  Windows 10
* **Graphics Tablet**:  Laptop/tablet

**Comment by turtletooth** [comment]
What hardware did you have this on?  I used a surface tonight with Toonz Raster levels and undid plenty and didn't notice this.

**Comment by manongjohn** [comment]
It appears to be specific to my laptop-tablet (Thinkpad Yoga) at least.

I did some additional testing on the same laptop with an attached Wacom tablet and it didn't have this issue. I checked Krita and it doesn't have this problem either.

I never noticed this before. I'm guessing OT is getting some events in a strange order and not handling it properly. Will need to look more into it when I get a chance.

**Comment by manongjohn** [comment]
I discovered what was causing this.  I was getting duplicate tablet events, 1 from the laptop's tablet   drivers and 1 from the Wacom driver for my external drawing tablet.  OT doesn't handle duplicate  back-to-back requests correctly.  

This would likely occur for anyone that may use a separate drawing tablet with a tablet enabled laptop as they'd likely have 2 sets of tablet drivers installed and running.  Work around is to disable the external tablet's drivers when its not being used.

I do have a simple fix for this in case it happens again: ignore immediate repeated request when starting to draw.

Unfortunately, I found another major problem as a result of this setup.  When I enable Pressure on the OT brushes, as soon as I touch the screen, it starts expanding the point outward, covering the entire canvas.  Looking to fix that as well.
